compose/src/main/java/com/bottlerocketstudios/compose/repository/RepositoryBrowserScreen.kt

```diff
@@ -188,3 +180,12 @@ private fun PreviewOuterScreenContent() {
         )
     }
 }
+
+data class RepositoryBrowserScreenState(
+    val path: State<String>,
+    val itemCount: State<Int>,
+    val repositoryItems: State<List<RepositoryItemUiModel>>,
+    val onRepositoryItemClicked: (RepositoryItemUiModel) -> Unit
+)
+
+private const val DEFAULT_ITEM_COUNT = 3
```

Comment:
@Josh-Gebbeken - Since this value needs to stay in sync with the list<RepositoryItemUiModel> it may be better to val that list a constant and then get the size of the list.  This would avoid maintaining this constant.